Tests/EntitlementsSpec.swift

```diff
@@ -0,0 +1,365 @@
+            }
+        }
+        
+        describe("FeatureFlagsService") {
+            var auth: Auth!
+            var featureFlagsService: FeatureFlagsService!
+            
+            beforeEach {
+                KindeSDKAPI.configure()
+                auth = KindeSDKAPI.auth
+                featureFlagsService = auth.featureFlags
+            }
+            
+            describe("getFeatureFlags") {
+                it("returns empty dictionary when no feature flags claim exists") {
+                    let flags = featureFlagsService.getFeatureFlags()
+                    expect(flags).to(beEmpty())
+                }
+                
+                it("returns feature flags dictionary when claim exists") {
+                    let flags = featureFlagsService.getFeatureFlags()
+                    expect(flags).to(beAKindOf([String: Any].self))
+                }
+            }
+            
+            describe("isFeatureEnabled") {
+                it("returns false for non-existent feature flag") {
+                    let result = featureFlagsService.isFeatureEnabled(flag: "non_existent_flag")
+                    expect(result).to(equal(false))
+                }
+                
+                it("returns default value for non-existent feature flag") {
+                    let result = featureFlagsService.isFeatureEnabled(flag: "non_existent_flag", defaultValue: true)
+                    expect(result).to(equal(true))
+                }
+            }
+            
+            describe("getFeatureFlag") {
+                it("returns default value for non-existent feature flag") {
+                    let result = featureFlagsService.getFeatureFlag(flag: "non_existent_flag", defaultValue: "default")
+                    expect(result).to(equal("default"))
+                }
+                
+                it("handles type conversion correctly") {
+                    let result = featureFlagsService.getFeatureFlag(flag: "test_flag", defaultValue: 42)
+                    expect(result).to(beAKindOf(Int.self))
+                }
+            }
+        }
+        
+        describe("ClaimsService") {
+            var auth: Auth!
+            var claimsService: ClaimsService!
+            
+            beforeEach {
+                KindeSDKAPI.configure()
+                auth = KindeSDKAPI.auth
+                claimsService = auth.claims
+            }
+            
+            describe("getClaim") {
+                it("returns nil for non-existent claim") {
+                    let result = claimsService.getClaim(forKey: "non_existent_claim")
+                    expect(result).to(beNil())
+                }
+                
+                it("returns claim value when claim exists") {
+                    // This would require a mock token with specific claims
+                    // TODO: Add proper test fixtures with mocked tokens
+                    pending("Requires mock token with claims")
```

Comment:
```suggestion
                    pending("Requires mock token with claims") {}
```

`pending` requires a closure.